packages/service-library/tests/long_running_tasks/test_long_running_tasks_task.py

```diff
@@ -181,8 +181,8 @@ async def not_unique_task(task_progress: TaskProgress):
 
 
 def test_get_task_id():
-    obj1 = TasksManager._create_task_id("")  # noqa: SLF001
-    obj2 = TasksManager._create_task_id("")  # noqa: SLF001
+    obj1 = TasksManager.generate_task_id("")  # noqa: SLF001
```

Comment:
TIP: use `faker.word()` to create task_names

Reply:
```python
def test_generate_task_id_is_not_idenpotent(faker: Faker):
    task_name = faker.word()
    for _ in range(A_FEW_TIMES):
        assert TaskManager.generate_task_id(task_name) !=TaskManager.generate_task_id(task_name)
```

Moreover. i would generally avoid having empty task_names i.e. add in generator an assert

Reply:
done